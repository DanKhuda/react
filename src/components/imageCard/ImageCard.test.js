import React from 'react';
import ImageCard  from "./";
import { render } from 'enzyme';

describe('ImageCard ', () => {
    const imageMock = {
        id: '123',
        alt_description: 'Alt Desc',
        urls:{
            small: 'somePng.jpg',
        }
    };

    it('should render correctly', () => {
        const wrapper = render(<ImageCard image={imageMock} />);
        expect(wrapper).toMatchSnapshot();
    });

});