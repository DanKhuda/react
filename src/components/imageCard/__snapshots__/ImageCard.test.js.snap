// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCard  should render correctly 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "images-grid__item",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "images-grid__link",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "alt": "Alt Desc",
              "class": "image",
              "src": "somePng.jpg",
            },
            "children": Array [],
            "name": "img",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "alt": undefined,
              "class": undefined,
              "src": undefined,
            },
            "x-attribsPrefix": Object {
              "alt": undefined,
              "class": undefined,
              "src": undefined,
            },
          },
        ],
        "name": "a",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
